### PR TITLE
Add script to export last X full days of tracks

### PR DESCRIPTION
**Pull request by apulverizer**
Need to:
- [x] Add Readme
- [ ] Add tests

### CODE REVIEW

File: scripts/export_tracks.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    logger.info("Authenticating...")
+    # Authenticate to ArcGIS Online
+    gis = GIS(arguments.org_url,
+              username=arguments.username,
+              password=arguments.password,
+              verify_cert=not arguments.skip_ssl_verification)
+    if gis.properties['isPortal']:
+        raise Exception("Export is not supported for the location tracking service with ArcGIS Enterprise")
+    logger.info("Exporting...")
+    tracks_item = gis.content.get(args.tracks_item)
```
Comment (niravi) [inline scripts/export_tracks.py:67]: Add handling if item is not found

File: scripts/export_tracks.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                                              "id": 0,
+                                              "where": f"location_timestamp <= '{end_date.in_tz('UTC').to_datetime_string()}' AND "
+                                                       f"location_timestamp >= '{start_date.in_tz('UTC').to_datetime_string()}'"
+                                          }
+                                      ]
+                                  }
+                                  )
+    logger.info("Downloading...")
+    # Download the CSV file
+    save_path = os.path.abspath(args.output_directory)
```
Comment (niravi) [inline scripts/export_tracks.py]: add handling if output dir is not found or if download is unsuccessful

**Review by niravi — APPROVED** [review]

File: scripts/export_tracks.py (hunk 1 above)
Comment (apulverizer) [reply scripts/export_tracks.py:67]: Done

Comment (apulverizer) [reply scripts/export_tracks.py:67]: Done

File: scripts/export_tracks.py (hunk 2 above)
Comment (apulverizer) [reply scripts/export_tracks.py]: I added a check for the directory. The API will raise an exception if the download/export fails.

**Review by niravi — APPROVED** [review]

**Pull request merged**